src/Cache/FilesystemCache.php

```diff
@@ -76,6 +76,14 @@ public function write(string $key, string $content): void
         throw new \RuntimeException(\sprintf('Failed to write cache file "%s".', $key));
     }
 
+    public function remove(string $name, string $cls): void
+    {
+        $key = $this->generateKey($name, $cls);
+        if (is_file($key) && !@unlink($key)) {
```

Comment:
guards against concurrent unlinks?
```suggestion
        if (!@unlink($key) && file_exists($key)) {
```

Reply:
👍 Done